packages/venia-ui/lib/components/Checkout/checkoutButton.css

```diff
@@ -1,4 +1,5 @@
 .icon {
     composes: root from '../Icon/icon.css';
     --stroke: white;
+    margin-top: -2px;
```

Comment:
The icon is very bottom heavy, so moving it up slightly improves the visual appearance of the checkout button.